daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -232,10 +232,10 @@ function Change-InstanceProperties {
   # Change time zone to Coordinated Universal Time.
   Run-Command tzutil /s 'UTC'
 
-  # Not supported on 6.1 client, but is supported on 6.1 server
+  # Supported on 6.1 client and 6.1 server with SP1 installed only
   $pn_path = 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion'
   $pn = (Get-ItemProperty -Path $pn_path -Name ProductName).ProductName
-  if ($pn -notlike '*Windows 7*') {
+  if (($pn -notlike '*Windows 7*') -or ($pn -notlike '*Windows Server 2008*')) {
```

Comment:
I'd suggest the following so we don't exclude 2008R2 SP1. The only way to determine the service pack level is the build number. I found this list of Windows [build versions](https://www.gaijin.at/en/infos/windows-version-numbers) and the 2008R2 SP1 build number is 7601.

` if (($pn -notlike '*Windows 7*') -or ($pn -notlike '*Windows Server 2008*' -and [System.Environment]::OSVersion.Version.Build -lt 7601)) {`